### ISSUE TITLE
Return values should be placed on same line as return statement

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#531]
We should report a violation in case a return value is placed on another line than the return statement.

For example, following code should report a violation:
```c#
return
          abc;
```
while following code should NOT report a violation:
```c#
return abc;
```
